templates/element/intermediate.php

```diff
@@ -2,22 +2,54 @@
 $session = $this->request->getSession();
 $projects = $session->read('Projects');
 ?>
+<div class="home-selector <?= $visible ?>">
+    <div class="card card-plain">
+        <div class="card-header">
+            <h1 class="card-title">Selecciona uno de tus proyectos</h1>
+        </div>
 
-<div class="home-selector visible">
-    <h1>Bienvenido a Colmena</h1>
+        <div class="card-body">
+            <div class="row">
+                <?php
+                foreach ($projects as $project) {
+                ?>
+                    <div class="col">
+                        <div class="card">
+                            <div class="card-body">
+                                <div class="project-title">
+                                    <h2>
+                                        <?= $project['name'] ?>
+                                    </h2>
+                                </div>
+                                <p class="project-description">
+                                    <?= $project['description'] ?>
+                                </p>
 
-    <?php foreach ($projects as $project) {
-    ?>
-        <div class="project" data-project=<?= $project['id'] ?>>
-            <p><?= $project['name'] ?></p>
+                                <button class="button project-button" data-project="<?= $project['id'] ?>">
+                                    Ver projecto
+                                </button>
+                            </div>
+                        </div>
+                    </div>
+                <?php
+                }
+                ?>
+                <div class="col">
+                    <a class="card card-plain border add-project" href="/admin/academical-manager/projects/add">
+                        <div class="card-body">
+                            <i class="fa fa-plus" aria-hidden="true"></i>
+                            <h5 class=" text-secondary"> Nuevo proyecto </h5>
+                        </div>
+                    </a>
+                </div>
+            </div>
         </div>
-    <?php
-    } ?>
+    </div>
 </div>
 
 <script>
     $(function() {
-        $('.project').click(function() {
+        $('.project-button').click(function() {
             $.ajax({
                 method: 'POST',
```

Comment:
Should this be here or in a separate function?

Reply:
Maybe functions in the intermediate template (which is the one where you select which of your projects you would like to manage) should be in the functions.js. It will be considered in the next PR if you approve that